src/bodypart.cpp

```diff
@@ -65,7 +65,7 @@ std::string body_part_name (body_part bp)
     case bp_foot_r:
         return _("right foot");
     default:
-        return _("appendix");
+        return _("appendage");
```

Comment:
Created issue to address this. https://github.com/CleverRaven/Cataclysm-DDA/issues/16133